doc/tutorials/dune-package-management/index.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Marek Kubica
+orphan: true
```

Comment:
the tutorial is linked in a toctree, so it's not an orphan. you can remove this line (sphinx doesn't warn about this)

Reply:
It was an orphan at first but then I realized that without linking it up to the toctree it wasn't really feasible to jump between sections. Fixed.